docs/UserGuide.md

```diff
@@ -140,11 +140,54 @@ Examples:
 * `delmod m/CS2103` deletes the existing module with code `CS2103` from FaculType.
 * `delmod m/CS2102` deletes the existing module with code `CS2102` from FaculType.
 
-### Clearing all entries : `clear`
+### Finding modules : `findmod`
 
-Clears all entries from the FaculType.
+Finds all  modules whose code or name contain the given keyword.
 
-Format: `clear`
+Format : `findmod KEYWORD [MORE_KEYWORDS]`
+
+* The search is insensitive. e.g. `cs2103` will match `CS2103` in FaculType.
+* Partial words will be matched. e.g. `database` will match `Database Systems` in FaculType.
+* Modules matching at least one keyword will be returned (i.e. “OR” search)
+  e.g. `CS210 algorithms` will return `CS2103`, `CS2100`, `Data Structures and Algorithms`.
+* The order of the keyword does not matter. e.g. `Statistics and Probability` will match  `Probability and Statistics`.
+
+Examples :
+
+* `findmod programming` returns `Programming Methodology I`, `Programming Methodology II`
+* `findmod CS11 security` returns `CS1101S`, `Computer Security`
+* `findmod cs210` returns `CS2100`, `CS2102`, `CS2103`
+
+### Assigning a professor to a module: `assign`
+
+Assigns a professor to a module.
+
+Format: `assign INDEX m/MODULE_CODE [MORE MODULE_CODES]`
+* Assigns all the `MODULE_CODE` to the contact at the specified `INDEX`. All of the `MODULE_CODE` **must exist** in FaculType in the first place.
```

Comment:
Hmm for this part is it better to have say "Assigns the contact/instructor at the specified index to the module code"? 🤔

Reply:
I have made some changes. Can you check it again? thanks.